scipy/spatial/ckdtree.pyx

```diff
@@ -478,6 +498,10 @@ cdef public class cKDTree [object ckdtree, type ckdtree_type]:
         np.ndarray               boxsize_data
         np.float64_t             *raw_boxsize_data
 
+    property size:
```

Comment:
We should probably also expose a count of leafnodes as a public attribute, which we get by counting them in ```_post_init```. We also need to fix the pickling to include the new attributes. If we set both their values in ```_post_init``` the pickling would work without including them. We could actually simplify the pickling of other attributes this way too.